src/test/resources/reference.conf

```diff
@@ -43,6 +43,14 @@ googleServices {
   }
 }
 
+schemaLock {
+  lockSchemaOnBoot = false
+  recheckTimeInterval = 5
+  maxTimeToWait = 60
+  instanceId = "sam-local"
```

Comment:
Curious to see what value(s) this gets in production.

Reply:
thanks for bringing this to my attention because it caused me to look into it further which necessitated some devops work.

see Jackie's commit in https://github.com/broadinstitute/firecloud-develop/pull/1010 for the relevant changes